cmd/agent/install_script.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
```

Comment:
It would be great to use this script in the integration (molecule) tests, because this is what the user will use install StackState

Reply:
see above